driver-core/src/main/com/mongodb/client/model/expressions/IntegerExpression.java

```diff
@@ -16,6 +16,9 @@
 
 package com.mongodb.client.model.expressions;
 
+/**
+ * Expresses a numeric integer value.
```

Comment:
Isn't _"numeric integer"_, _"logical boolean"_ akin to electrical electronics?

Reply:
Discussed in Zoom, decided to call them Platonic integer and Platonic boolean :trollface: